core/src/main/resources/static/css/mobile.css

```diff
@@ -258,11 +261,12 @@ div.oneToSeven {
     font-size: .6em;
 }
 
-.navbar-nav {
-    max-width: 700px;
+@media(max-width: 1067px) {
```

Comment:
my goodness how this mobile.css file makes my teeth ache and my head pound.  An epic refactor must happen.